notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb

```diff
@@ -0,0 +1,754 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    # but many of the periods are extremely long; too long to be observed by TESS.
</pre><p>I'd say "unlikely to be observed by TESS". There's a <em>chance</em> that TESS <em>happened</em> to catch them at the right moment, it's just not very likely given the long period.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>